eslint.json

```diff
@@ -27,12 +30,21 @@
         "SwitchCase": 1
       }
     ],
+    "jsdoc/check-param-names": 1,
+    "jsdoc/check-tag-names": 1,
+    "jsdoc/newline-after-description": 1,
+    "jsdoc/no-undefined-types": 1,
```

Comment:
This is not needed. It is conflicting with `no-unused-vars`.

> When enabling this rule, types in jsdoc comments will resolve as used variables, i.e. will not be marked as unused by no-unused-vars.

https://github.com/gajus/eslint-plugin-jsdoc#no-undefined-types

Reply:
We don't need this rule.